core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -152,6 +180,13 @@ protected void checkVersion(long clientVersion) throws IOException {
     }
   }
 
+  /**
+   * @return If the client should skip version check service
+   */
+  protected boolean skipVersionCheck() {
+    return false;
+  }
```

Comment:
`BlockWorkerClient` doesn't check versions as alluxio workers don't hold version service. This provides a switch for inheriting classes to skip version checks

Reply:
Yeah that's true. I finally realized `BlockWorkerGrpc`'s service type is actally `FILE_SYSTEM_WORKER_WORKER_SERVICE`.

Reply:
I added this method because the original `DefaultBlockWorkerClient` doesn't check version though.

Reply:
I would give it a new name like BLOCK_WORKER_CLIENT_SERVICE. And could we just use a constant number in getServiceVersion to avoid adding a new API? your call